.github/workflows/ci.yml

```diff
@@ -44,16 +46,9 @@ jobs:
           - 17
     timeout-minutes: 45
     steps:
-      - uses: actions/checkout@v2
+      - uses: nineinchnick/trino/.github/actions/checkout-and-setup@7402e992a8e308da2bf48826428ef8bbee7cd94f
```

Comment:
If the general idea will be approved, we can merge this action in a separate PR and then reference the SHA from the master branch.

Reply:
Create an `actions` repo in the `trinodb` org and reference it from there.

Reply:
I created the repo in my account for now, we can move it when this is approved.